res/Sectors/M1105/kidunal.txt

```diff
@@ -0,0 +1,241 @@
+0311 Klootkolul           C6136S6-9 Ic Na Ni              { -1 } (A53-2) [7578] - M  - 403   HvFd M7 V M1 V       
+0315 Rulyass              B652948-C Hi Po                 {  3 } (B86-2) [8C6C] - N  - 603   CsHv M0 V
+0413                      E120000-0 Ba De Po              { -3 } (600-2) [0000] - -  - 024   CsHv G7 V M0 V       
+0415                      C341656-8 He Ni Po              { -2 } (650+2) [1419] - N  - 600   Na   K4 V            
+0513                      E554558-5 Ag Ni                 { -2 } (740+4) [6338] - -  - 813   CsHv M0 V M3 V       
+0514                      E610140-7 Lo                    { -3 } (801+4) [1156] - -  - 703   CsHv K6 V            
+0611                      A758832-A Ph Pa                 {  2 } (D7E+2) [4A3A] - -  - 623   CsHv M1 V         
+0613                      B777622-A Ag Ni                 {  2 } (B55-3) [4878] - N  - 504   Na   G6 V            
+1113 Kidunal              E784985-6 Hi Pr                 { -1 } (B89+1) [7835] - -  - 823   NaXX M8 V            
+2412                      B769611-A Ni Ri                 {  2 } (A57+1) [685A] - -  - 302   NaXX M0 V            
+2418                      E456416-6 Ni Pa                 { -3 } (B31-1) [1136] - -  - 113   NaXX F2 IV           
+2713 Vuam                 E351954-8 Hi Po                 { -1 } (B85-1) [A868] - -  - 813   NaXX M3 V            
+2720                      E200000-0 Ba Va                 { -3 } (600+2) [0000] - -  - 002   NaXX M8 V          
+3111 Pixa                 D481A98-A Hi                    {  1 } (898-2) [9B48] - N  - 200   NaXX K7 V M2 V       
+3115                      D465646-7 Ag Ni Ri              { -1 } (754+3) [754B] - -  - 810   NaXX A5 V            
+0121 Aftepusk             D553946-7 Hi Po                 { -1 } (988-2) [B849] - -  - 803   Na   M4 III          
+0128 Kroweso              C441342-9 He Lo Po              { -1 } (B21-2) [7288] - N  - 901   NaXX M5 V   
+0221                      E000000-0 As Ba Va              { -3 } (B00-1) [0000] - -  - 023   NaXX M2 V M1 V       
+0222                      C66A624-9 Ni Ri Wa              {  0 } (A54-2) [A63A] - N  - 313   NaXX K5 V            
+0223                      D243412-7 Ni Po                 { -3 } (331-4) [2146] - -  - 703   Hg   M9 V M0 V       
+0225                      C386514-5 Ag Ni Pr              { -1 } (A44+2) [3452] - N  - 503   Hg   M8 V M3 V       
+0227                      E110000-0 Ba                    { -3 } (700-1) [0000] - -  - 004   NaXX M1 V            
+0229                      E500000-0 Ba Va                 { -3 } (800-5) [0000] - -  - 005   NaXX M5 V            
+0230                      C150446-8 De Ni Po              { -2 } (731-3) [1229] - N  - 712   NaXX M9 V            
+0322                      B5333B6-7 Lo Po Pz              { -1 } (A21+1) [6218] - N  A 822   NaXX G4 V M9 V       
+0323 Yoceklool            C666944-8 Hi Pr Ga              {  0 } (F85-1) [9977] - N  - 633   Hg   M9 V            
```

Comment:
This should have the 'Cx' code (Capital)

Reply:
I changed the UWP of the high-tech world in this polity, so I've made the captial as it make more sense (to me)